ntbs-service/Pages/Notifications/NotificationModelBase.cs

```diff
@@ -37,7 +39,10 @@ public abstract class NotificationModelBase : PageModel, IWithNotificationBanner
 
         [BindProperty(SupportsGet = true)]
         public int NotificationId { get; set; }
-
+        public string TopLevelBreadcrumbUrl { get; set; }

```

Comment:
ups, i don't need these anymore will review (and 2 below)

Reply:
good catch!